jina/executors/indexers/vector.py

```diff
@@ -363,6 +364,12 @@ def query(self, vectors: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np
         return indices, dist
 
     def build_advanced_index(self, vecs: 'np.ndarray') -> 'np.ndarray':
+        """
+        Build advanced index structure based on in-memory numpy ndarray, e.g. graph, tree, etc.
+
+        :param vecs: the raw numpy ndarray
+        :return:
```

Comment:
return details

Reply:
This method seems not implemented. I cannot how it actually works.